coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/presenter/CoachmarkPresenter.kt

```diff
@@ -31,7 +31,7 @@ internal class CoachmarkPresenter(private val view: CoachmarkViewInterface) {
 
         stepReferenced.view.let {
             val isBodyMoreBottom = bodyGlobalRect.bottom < stepReferenceGlobalRect.bottom
-            if (isBodyMoreBottom || !it.getLocalVisibleRect(bodyGlobalRect) || bodyGlobalRect.height() < it.height) {
+            if (isBodyMoreBottom || !it!!.getLocalVisibleRect(bodyGlobalRect) || bodyGlobalRect.height() < it!!.height) {
```

Comment:
```suggestion
            if (isBodyMoreBottom || !it.getLocalVisibleRect(bodyGlobalRect) || bodyGlobalRect.height() < it.height) {
```

si previamente usamos el `view?.let{}` it ya es != null